README.md

```diff
@@ -47,3 +47,10 @@ this Program grant you additional permission to convey the resulting work.
 Library | Group ID | Artifact ID | SPDX License ID
 :-- | :-- | :-- | :--
 Jakarta Mail | com.sun.mail | jakarta.mail | GPL-2.0-only
+
+## Open Source Delevoper Licence for YourKit Profiler [![YourKit logo](https://www.yourkit.com/images/yklogo.png)]
+
+YourKit supports open source projects with innovative and intelligent tools for monitoring and profiling Java and .NET applications.
+YourKit is the creator of <a href="https://www.yourkit.com/java/profiler/">YourKit Java Profiler</a>, <a href="https://www.yourkit.com/.net/profiler/">YourKit .NET Profiler</a>, and <a href="https://www.yourkit.com/youmonitor/">YourKit YouMonitor</a>.
+
+YourKit grants the TripleA development project an open source licence for YourKit Java Profiler.
```

Comment:
I somewhat feel all we really need is the last sentence. I know this was the example text, but it was an example : )
WDYT? 
(side-comment) Can use markdown sytnax for the links to make it a bit more concise in the markdown.